server/modules/record/api/recordApi.js

```diff
@@ -153,6 +174,17 @@ export const init = (app) => {
     }
   })
 
+  app.get('/survey/:surveyId/records/summary/count', requireRecordListViewPermission, async (req, res, next) => {
+    try {
+      const { surveyId, cycle, search } = Request.getParams(req)
+
+      const count = await RecordService.countRecordsBySurveyId({ surveyId, cycle, search })
+      res.json(count)
+    } catch (error) {
+      next(error)
+    }
+  })
+
```

Comment:
at this point /survey/:surveyId/records/count should be removed, or just changed into /survey/:surveyId/records/summary/count, otherwise we have 2 endpoints doing the same thing